### PR TITLE
Feature/functional board post list

### PR DESCRIPTION
**Pull request by LimbertVargas**

### CODE REVIEW

File: trello/features/functional/BoardFunctional.feature [hunk 1]
```diff
@@ -71,8 +71,26 @@ Feature: Board
     Then Should return status code 200
     And Validates response body
     And Validates schema
-    And Sets a "DELETE" request to "/boards/"
-      | key | value         |
-      | id  | board_test.id |
+    And Sets a "DELETE" request to "/boards/(id)"
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardFunctional.feature]: this (id) part of code improve to (boardObject.id)

File: trello/features/functional/BoardFunctional.feature [hunk 2]
```diff
@@ -71,8 +71,26 @@ Feature: Board
...
+    And Sends request
+    And Should return status code 200
+
+
+  # Created by Limbert Vargas at 1/29/2020
+  Scenario: Creates a list in board
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardFunctional.feature:80]: You could add a little description of this scenario

File: trello/features/functional/BoardFunctional.feature [hunk 3]
```diff
@@ -71,8 +71,26 @@ Feature: Board
...
+    Given Sets a "POST" request to "/boards/"
+      | key  | value    |
+      | name | newBoard |
     And Sends request
     And Should return status code 200
+    And Saves response as "board_test"
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardFunctional.feature]: change the name of **board_test** to (boardObject)

File: trello/features/functional/BoardFunctional.feature [hunk 4]
```diff
@@ -71,8 +71,26 @@ Feature: Board
...
+    And Sets a "POST" request to "/boards/(id)/lists"
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardFunctional.feature]: You could improve this (id) to (boardObject.id)

File: trello/features/functional/BoardFunctional.feature [hunk 5]
```diff
@@ -71,8 +71,26 @@ Feature: Board
...
+      | key     | value            |
+      | idBoard | (board_test.id) |
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardFunctional.feature]: chech the identation of this lines

File: trello/features/functional/BoardFunctional.feature [hunk 6]
```diff
@@ -71,8 +71,26 @@ Feature: Board
...
+    When Sends request
+    Then Should return status code 200
+    And Validates response body
+    And Validates schema
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardFunctional.feature]: improve this line of code specifying the file json_board

File: trello/features/functional/BoardFunctional.feature [hunk 7]
```diff
@@ -71,8 +71,26 @@ Feature: Board
...
+    And Sets a "DELETE" request to "/boards/(id)"
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardFunctional.feature]: improve this part of code (id) to (boardObject.id)

File: trello/features/functional/BoardFunctional.feature [hunk 8]
```diff
@@ -71,8 +71,26 @@ Feature: Board
...
+    And Sends request
+    And Should return status code 200
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardFunctional.feature]: ad a new line at end of the code lines

File: trello/features/functional/BoardFunctional.feature [hunk 9]
```diff
@@ -71,8 +71,26 @@ Feature: Board
     Then Should return status code 200
     And Validates response body
     And Validates schema
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardFunctional.feature]: improve this line of code specifying the json schema file

File: trello/features/functional/BoardFunctional.feature [hunk 10]
```diff
@@ -50,7 +50,7 @@ Feature: Board
       | id  | organization_test.id |
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardFunctional.feature:50]: Is it correct this id value in this line code? maybe this id can go joined with url value, so this is step is to delete
check please

Comment (LimbertVargas) [reply trello/features/functional/BoardFunctional.feature:50]: this part is Enrique's code

File: trello/features/functional/BoardFunctional.feature [hunk 11]
```diff
@@ -71,8 +71,27 @@ Feature: Board
...
-    And Sets a "DELETE" request to "/boards/"
-      | key | value         |
-      | id  | board_test.id |
+    And Sets a "DELETE" request to "/boards/(boardObject.id)"
+    And Sends request
+    And Should return status code 200
+
+
+  # Created by Limbert Vargas at 1/29/2020
+  # This scenario allows to create a list on an existing board
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardFunctional.feature]: this description goes after the scenario line

File: trello/features/functional/BoardFunctional.feature [hunk 12]
```diff
@@ -71,8 +71,27 @@ Feature: Board
...
+  Scenario: Creates a list in board
+    Given Sets a "POST" request to "/boards/"
+      | key  | value    |
+      | name | newBoard |
+    And Sends request
+    And Should return status code 200
+    And Saves response as "(boardObject)"
+    And Sets a "POST" request to "/boards/(boardObject.id)/lists"
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardFunctional.feature:89]: check if this post request is the correct and the endpoint

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Pull request merged**